server/dearmep/api/authtoken.py

```diff
@@ -6,17 +6,16 @@
 from datetime import datetime, timezone
 from typing import Annotated
 
+import jwt
 from fastapi import Depends, HTTPException, status
-from fastapi.security import OAuth2PasswordBearer
-from jose import JWTError, jwt
+from fastapi.security import HTTPAuthorizationCredentials, HTTPBearer
 from pydantic import ValidationError
 
 from ..config import Config
 from ..models import JWTClaims, JWTResponse, PhoneNumber
 
 
-oauth2_scheme = OAuth2PasswordBearer(
-    tokenUrl="/api/v1/number-verification/request",
+oauth2_scheme = HTTPBearer(
```

Comment:
Because the flow with SMS authentication doesn't really fit a typical password-flow. It's also much nicer now to use the swagger UI to log in. Simply copy the JWT into the login form.

Reply:
That makes sense, thanks :)